Content.Server/Kitchen/EntitySystems/MicrowaveSystem.cs

```diff
@@ -255,7 +262,34 @@ private void OnBreak(EntityUid uid, MicrowaveComponent component, BreakageEventA
         {
             component.Broken = true;
             SetAppearance(component, MicrowaveVisualState.Broken);
+
+           if (TryComp<ActiveMicrowaveComponent>(uid, out var activeMicrowareComponent))
+            {
+                if (activeMicrowareComponent.Haywire)
+                {
+                    _audio.PlayPvs(component.FoodDoneSound, component.Owner, AudioParams.Default.WithVolume(-1));
+                    var random = new RobustRandom();
+
+                    // Make it look like the microwave exploded
+                    var machineFrame = Spawn("MachineFrameDestroyed", new Robust.Shared.Map.EntityCoordinates(uid, Vector2.Zero));
+                    Comp<TransformComponent>(machineFrame).Anchored = false;
+                    _throwingSystem.TryThrow(machineFrame, new Vector2(random.NextFloat()*20-10, random.NextFloat()*20-10), 10);
+
+                    // This doesn't seem to be spawning the microwave board, maybe someone else can figure out why?
+                    // It gets created and then instantly deleted, even with godmode and explosive resistance
```

Comment:
You're spawning it as a child of the microwave.

Reply:
Ahhhhh okay, I didn't realize Spawn worked like that, thanks!

Reply:
Yeah, that totally fixed it, thanks! :D